### PR TITLE
Update spacer test cases

### PR DESCRIPTION
**Pull request by jbinda**

### CODE REVIEW

File: test-cases/gutenberg/spacer.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+##### TC005
+
+### Spacer in horizontal layout works as expected
+
+-   Add a `Spacer` block
+-   Rotate the device to horizontal layout
+-   Check if `Spacer` has proper height according to settings
+
+--------------------------------------------------------------------------------
```
Comment (lukewalczak) [inline test-cases/gutenberg/spacer.md]: More comments:

* I would like to have test case similar to TC002, but the value is set using input.
* If you will change title remember to change them in `test-suites/gutenberg/sanity-tests.md` as well

File: test-cases/gutenberg/spacer.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+##### TC001
+
+### Spacer is rendered without crash
+
+-   Add a `Spacer` block
+-   Check if `Spacer` is rendered
+-   Check if app do not crash when open settings
```
[the remark this answers is not in the document]
Comment (jbinda) [reply test-cases/gutenberg/spacer.md]: done

File: test-cases/gutenberg/spacer.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Expected look:  
+![Spacer](../resources/spacer.png)
+
+--------------------------------------------------------------------------------
+
+##### TC002
+
+### Settings: Adjusting spacer height with slider
```
[the remark this answers is not in the document]
Comment (jbinda) [reply test-cases/gutenberg/spacer.md]: I think it's looks better

File: test-cases/gutenberg/spacer.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+-   Add a `Spacer` block
+-   Check if `Spacer` block is rendered with default height of `100px`
+-   Change slider value under `Spacer` settings and measure if `Spacer` block has height setted via slider
```
[the remark this answers is not in the document]
Comment (jbinda) [reply test-cases/gutenberg/spacer.md]: implemented, thanks !

File: test-cases/gutenberg/spacer.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+![SpacerRanges](../resources/spacer-max-height.png)
+
+--------------------------------------------------------------------------------
+
+##### TC004
+
+### Settings: Check if height range extends when Spacer was saved in web version
+
+-   Add a `Spacer` block via web version
+-   Set `Spacer` height to more that `500px` and save the post
```
[the remark this answers is not in the document]
Comment (jbinda) [reply test-cases/gutenberg/spacer.md]: fixed

File: test-cases/gutenberg/spacer.md (hunk 1 above)
Comment (jbinda) [reply test-cases/gutenberg/spacer.md]: I have added TC for manipulate height with textinput and update titles in sanity-test.md

**Pull request merged**